workers/unity/Packages/com.improbable.gdk.core/Systems/WorkerDisconnectCallbackSystem.cs

```diff
@@ -5,6 +5,9 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     Enables users to add a callback onto the disconnection event from the ECS context.
```

Comment:
not necessarily the ECS context. they could also register to the callback from a MonoBehaviour. This is simply the system where the callback gets triggered

Reply:
I noticed that we expose a callback somewhere in the Worker abstraction as well - which makes me question if this is even still needed..